sync-template.sh

```diff
@@ -51,6 +51,8 @@ FILES=(
   '--copy-if-missing=pylint_extra_requirements.txt'
   '--copy=pylintrc'
   '--copy=tox.ini'
+  '--copy=.yamllint.yml'
+  '--copy-if-missing=.yamllint_custom.yml'
```

Comment:
By the way, I made a PR for this - https://github.com/richm/auto-maintenance/pull/1 - maybe I should have advertized it more loud.

Reply:
Yeah, not sure why I didn't see this earlier

Reply:
If you accept this suggestion https://github.com/linux-system-roles/network/pull/178#discussion_r399975678 it will need to change.